lib/board.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../lib/player.rb'
+class Board
+  attr_reader :game_board, :slots_taken
+
+  def initialize
+    @game_board = [
+      [' ', ' ', ' '],
+      [' ', ' ', ' '],
+      [' ', ' ', ' ']
+    ]
+    @slots_taken = []
+  end
+
+  def show_board
+    @game_board.each do |row|
+      row.each { |col| print %(| #{col} | ) }
+      puts %()
```

Comment:
From Pathwright:
![image](https://user-images.githubusercontent.com/2739245/91670026-4c7ba680-eadf-11ea-9b74-0348e4c57c4f.png)

- Please avoid `puts` and `print` everywhere except in `main.rb`. To achieve this, you may return string values from your `Board` class, and then use `puts` in `main.tb` to display the returned values.